enos/modules/docker_boundary/init.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+# This script initializes a postgres database to work with Boundary by spinning up a temporary
+# Boundary docker container pointed to the specified database and invoking `boundary database init`.
+#
+# This script must only output the JSON that comes from `boundary database init` as the output is
+# consumed by other scripts.
+
+TEST_BOUNDARY_IMAGE=$1
+TEST_DATABASE_ADDRESS=$2
+TEST_CONTAINER_NAME=boundary-init
+SOURCE=$(realpath $(dirname ${BASH_SOURCE[0]})) # get directory of this script
+
+docker run \
+    -d \
+    --name $TEST_CONTAINER_NAME \
+    -e "BOUNDARY_POSTGRES_URL=$TEST_DATABASE_ADDRESS" \
+    -e "SKIP_CHOWN=true" \
+    --cap-add IPC_LOCK \
+    --mount type=bind,src=$SOURCE,dst=/boundary/ \
+    --network e2e_network \
+    $TEST_BOUNDARY_IMAGE \
+    boundary database init -config /boundary/boundary-config.hcl -format json \
+    >/dev/null 2>&1 # suppresses output
+
+docker wait $TEST_CONTAINER_NAME >/dev/null 2>&1
+echo $(docker logs $TEST_CONTAINER_NAME)
+docker rm -v $TEST_CONTAINER_NAME >/dev/null 2>&1 || true
```

Comment:
Instead of starting a container in the background, waiting for it to finish, printing its logs and then remove it, why don't we let it run to completion in the foreground with `--rm`?
```suggestion
docker run \
    --rm \
    --name $TEST_CONTAINER_NAME \
    -e "BOUNDARY_POSTGRES_URL=$TEST_DATABASE_ADDRESS" \
    -e "SKIP_CHOWN=true" \
    --cap-add IPC_LOCK \
    --mount type=bind,src=$SOURCE,dst=/boundary/ \
    --network e2e_network \
    $TEST_BOUNDARY_IMAGE \
    boundary database init -config /boundary/boundary-config.hcl -format json \
    >/dev/null 2>&1 # suppresses output
```